app/models/rbac.rb

```diff
@@ -454,6 +454,8 @@ def self.search(options = {})
   def self.method_with_scope(ar_scope, options)
     if ar_scope == VmdbDatabaseConnection
```

Comment:
Aaron, I can do that. Still, I am not sure why VmdbDatabaseSetting causes the issue in the first place and was hoping there may be a resolution that does not require hard coding for a specific model.